igvc_navigation/launch/octomap.yaml

```diff
@@ -29,6 +29,7 @@ filters:
         enabled: true
         start_angle: -0.5 #Radians
         end_angle: 0.5
+        ray_start_distance: 1
```

Comment:
Shouldn't this value be set to be the same value as the minimum lidar range?

Reply:
Yeah, this is the minimum lidar range.